symphony/rules.mk

```diff
@@ -22,11 +22,21 @@
 
 include mk/subdir_pre.mk
 
-SYMPHONY_DIR := $(d)/
+SYMPHONY_DIR := $(d)
 SYMPHONY_MODS := utils orchestration client cli runtime runner local schemas
 
 SYMPHONY_PUBLICATION_REPO ?= testpypi
 
+TYPECHECK_JOBS := 2
+
+typecheck-%:
+	pytype --keep-going --config=$(SYMPHONY_DIR)$*/pyproject.toml --jobs $(TYPECHECK_JOBS)
+
+symphony-typecheck:
```

Comment:
Not very dramatic, but technically both targets should be listed as `.PHONY` below. :)